Mathlib/RingTheory/Ideal/Over.lean

```diff
@@ -430,3 +430,98 @@ lemma map_eq_top_iff {R S} [CommRing R] [CommRing S]
 end IsDomain
 
 end Ideal
+
+section ideal_lies_over
+
+variable (A : Type*) [CommSemiring A] {B : Type*} [Semiring B] [Algebra A B]
+  (P : Ideal B) (p : Ideal A)
+
+/-- The ideal obtained by pulling back the ideal `P` from `B` to `A`. -/
+abbrev Ideal.under : Ideal A := Ideal.comap (algebraMap A B) P
```

Comment:
I'm not entirely convinced that this is needed.

Reply:
I think that writing `comap (algebraMap A B) P` is too verbose, especially when the forms of `A` and `B` are quite complex. So I wrote this abbreviation.

Reply:
Fair enough. I like this in principle, but I'm a bit afraid if some automation wouldn't like it. Let's keep it for now.